lib/components/app/responsive-webapp.js

```diff
@@ -243,10 +246,13 @@ class RouterWrapperWithAuth0 extends Component {
             <Route exact path={ACCOUNT_PATH}>
               <Redirect to={TRIPS_PATH} />
             </Route>
+            <Route exact path={CREATE_ACCOUNT_PATH}>
+              <Redirect to={CREATE_ACCOUNT_TERMS_PATH} />
+            </Route>
             <Route
               // This route lets new or existing users edit or set up their account.
               component={UserAccountScreen}
-              path={[CREATE_ACCOUNT_PATH, ACCOUNT_SETTINGS_PATH]}
+              path={[`${CREATE_ACCOUNT_PATH}/:id`, ACCOUNT_SETTINGS_PATH]}
```

Comment:
`:id` is a bit confusing, since we're not dealing with an entity id, but rather a stage of account creation. There may be some weirdness with how these params get processed though, so this is non-blocking (especially if headaches are involved).

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/35eb848eeb28b878ba59d7fd07e01b18f92fa260.